bin/yaml/v.yaml

```diff
@@ -0,0 +1,12 @@
+compilers:
+  v:
+    type: ziparchive
+    dir: v-{name}
+    url: https://github.com/vlang/v/releases/download/weekly.{name}/v_linux.zip
+    folder: v
+    check_exe: v --version
+    after_stage_script:
```

Comment:
This is still present here...can you remove this `after_stage_script` please as it doesn't do anything. Thanks!

Reply:
The `export` command is still needed for the `v --version` check to work properly, since even this simple command needs readable access to a `VMODULES` directory currently.

Reply:
The `after_stage_script` is just a script that runs after staging: it is not persistent.. We need to set `check_env` to affect the checks.

I'll make the changes necessary and merge rather than cause another delay. Thanks!

Reply:
Ah ok, tysm!